doc/user/bgp.rst

```diff
@@ -31,9 +31,10 @@ be specified (:ref:`common-invocation-options`).
 
 .. option:: -l, --listenon
 
-   Specify a specific IP address for bgpd to listen on, rather than its default
+   Specify specific IP addresses for bgpd to listen on, rather than its default
```

Comment:
We need an example of how to specify multiple addresses for the end user.

Reply:
I added an example based on what I believe to be a typical use of it.